MoltenVK/MoltenVK/GPUObjects/MVKPipeline.mm

```diff
@@ -305,7 +312,7 @@
 				cmdEncoder->_renderingState.setFrontFace(_rasterInfo.frontFace, false);
 				cmdEncoder->_renderingState.setPolygonMode(_rasterInfo.polygonMode, false);
 				cmdEncoder->_renderingState.setDepthBias(_rasterInfo);
-				cmdEncoder->_renderingState.setDepthClipEnable( !_rasterInfo.depthClampEnable, false );
+				cmdEncoder->_renderingState.setDepthClipEnable( !_rasterInfo.depthClampEnable, false);
```

Comment:
```suggestion
				cmdEncoder->_renderingState.setDepthClipEnable(!_rasterInfo.depthClampEnable, false);
```

Reply:
One of my style quirks is that I deliberately try to include a space between `( !`  to call out the `!` during reading. Nothing worse than missing a "not" in any communication! 😉

 I've fixed this the other way, and added a matching space at the end.